modules/300-prometheus/docs/README.md

```diff
@@ -73,13 +73,20 @@ Deckhouse supports sending alerts using `Alertmanager`:
 
 <img src="../../images/300-prometheus/prometheus_monitoring.svg">
 
+##### WIP: new scheme for single datasource (testing_now)
+
+<img src="../../images/300-prometheus/prometheus_monitoring_new.svg">
```

Comment:
Let's make it official an remove trickster.

Reply:
in next iteration